config/mc_prod_setups/Run3_mu.yaml

```diff
@@ -20,11 +20,27 @@ points:
     mu_mixing: 0.112
   - mass: 2
     mu_mixing: 0.0135
-  - mass: 3
-    mu_mixing: 0.00379
+  # - mass: 3
+  #   mu_mixing: 0.00379
   - mass: 4
     mu_mixing: 0.00151
   - mass: 5
```

Comment:
Do we want to have some points between 5 and 6 to close the contour for the 2-sigma band? Same comment for Run2

Reply:
As discussed in the meeting, I will add a couple of mass points after 5 in case they are needed for Run2-Run3 combination.